test/user/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+set -e
+
+cd $(dirname $0)
+. ../util.sh
+
+docker compose up -d
+sleep 15 
+
+docker compose exec backup backup
+sudo cp -r $(docker volume inspect --format='{{ .Mountpoint }}' commands_archive) ./local
```

Comment:
This is questionable. What is commands_archive?

Reply:
This copies the files from the docker volume named `commands_archive` to the `./local` directory. The volume is defined here https://github.com/offen/docker-volume-backup/blob/2ac1f0cea4689466ded407699519c874bba31101/test/commands/docker-compose.yml#L51 (compose adds the prefix)